### PR TITLE
Added introductory and web-dev resources

### PR DESCRIPTION
**Pull request by shivams**
Added some intro to CS and web-dev resources. Will add more later.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,8 +13,15 @@
 
 Any resources that do not have a free version accessible (or require physical equipment other than a computer) are marked with a :heavy_dollar_sign:.
 
+## Intro to CS
+
+* :man_teacher: :movie_camera: [CS50 by David Malan](https://cs50.harvard.edu/college/2021/spring/)
```
Comment (emschwartz) [inline README.md]:
```suggestion
* :teacher: :movie_camera: [CS50 by David Malan](https://cs50.harvard.edu/college/2021/spring/)
```

Comment (emschwartz) [reply README.md]: Do you think it's better to provide that link or the one to EdX? https://www.edx.org/course/introduction-computer-science-harvardx-cs50x

Also, want to add a very brief description of what the course covers?

File: README.md [hunk 2]
```diff
@@ -13,8 +13,15 @@
...
+* :book: :heavy_dollar_sign: [Computer Science Distilled](https://www.goodreads.com/book/show/34189798-computer-science-distilled) - a small 180-page book providing a concise yet rigorous introduction to Computer Science
+
 ## Web Development
 
+* :man_teacher: :movie_camera: [CS50 Web by Brian Hu and David Malan](https://cs50.harvard.edu/web/2020/) (this is different than the vanilla CS50)
```
Comment (emschwartz) [inline README.md]:
```suggestion
* :teacher: :movie_camera: [CS50 Web by Brian Hu and David Malan](https://cs50.harvard.edu/web/2020/) (this is different than the vanilla CS50)
```

**Review by emschwartz — CHANGES_REQUESTED** [review]

**Comment by emschwartz** [comment]
Thanks for the additions!

File: README.md (hunk 1 above)
Comment (shivams) [reply README.md]: We can add the EdX link as an alternative link. And I will add a brief description as well.

Also, my markdown editor (Typora) was only showing `:man_teacher:` and not `:teacher:` in the drop-down list. But will make sure to use `:teacher:` in future to keep it gender neutral :smile:

**Comment by shivams** [comment]
@emschwartz I've added the descriptions, EdX link and fixed the emojis.

**Review by emschwartz — APPROVED** [review]

**Pull request merged**